inc/classes/class-genesis-featured-widget-amplified.php

```diff
@@ -915,41 +907,37 @@ public function form( $instance ) {
 		foreach ( $columns as $column => $boxes ) {
 			if ( 'col1' === $column ) {
 				echo '<div style="float: left; width: 250px;">';
-
 			} else {
 				echo '<div style="float: right; width: 250px;">';
 			}
 
 			foreach ( $boxes as $box ) {
-
 				echo '<div style="background: #f1f1f1; border: 1px solid #DDD; padding: 10px 10px 0 10px; margin-bottom: 5px;">';
 
 				foreach ( $box as $field_id => $args ) {
-
 					$class = $args['save'] ? 'gfwa-widget-control-save' : '';
 					$style = $args['requires'] ? ' style="' . gfwa_get_display_option( $instance, $args['requires'][0], $args['requires'][1], $args['requires'][2] ) . '"' : '';
 
 					switch ( $args['type'] ) {
-
 						case 'post_type_select':
 							echo '<p><label for="' . esc_attr( $this->get_field_id( $field_id ) ) . '">' . str_replace( esc_html( '<br />' ), '<br />', esc_html( $args['label'] ) ) . ':</label>
-								<select class="' . esc_html( $class ) . '" id="' . esc_attr( $this->get_field_id( $field_id ) ) . '" name="' . esc_attr( $this->get_field_name( $field_id ) ) . '">';
+								<select class="' . esc_html( $class ) . '" id="' . esc_attr( $this->get_field_id( $field_id ) ) . '" name="' . esc_attr( $this->get_field_name( $field_id ) ) . '">'; // WPCS: XSS ok.
 
-							$args       = array(
+							$args                               = array(
```

Comment:
What is happening with the spacing here? It looks off but maybe I’m missing something in the git review tab.

Reply:
I ran `phpcbf`, and this is what it did. It should be aligning the `=` on consecutive lines, but I agree - it looks off here, so a follow-up commit incoming imminently to make this sensible while still passing.

Reply:
Huh. I wonder if there is a version diff for the WPCS I’m using locally. When I used phpcbf I got the original spacing.